internal/installercache/installercache.go

```diff
@@ -183,33 +291,34 @@ func (i *Installers) evict() {
 		return nil
 	}
 
-	err := filepath.Walk(i.cacheDir, visit)
+	err := filepath.Walk(i.config.CacheDir, visit)
 	if err != nil {
 		if !os.IsNotExist(err) { //ignore first invocation where the cacheDir does not exist
-			i.log.WithError(err).Errorf("release binary eviction failed to inspect directory %s", i.cacheDir)
+			i.log.WithError(err).Errorf("release binary eviction failed to inspect directory %s", i.config.CacheDir)
 		}
 		return
 	}
 
-	//prune the hard links just in case the deletion of resources
-	//in ignition.go did not succeeded as expected
-	for idx := 0; idx < len(links); idx++ {
-		finfo := links[idx]
-		//Allow a grace period of 5 minutes from the link creation time
-		//to ensure the link is not being used.
-		grace := time.Now().Add(DeleteGracePeriod).Unix()
-		if finfo.info.ModTime().Unix() < grace {
-			os.Remove(finfo.path)
-		}
-	}
```

Comment:
Just FYI it's tough to review stuff like this where you both move and slightly change something.
Given that this PR is already pretty big and the changes you made in this case really a no-op I think it would have been better to just not change this at all.

Now that you've done it, it's fine, but just wanted to make you aware.

Reply:
I will bear this in mind, I'll try to keep things aligned so they 'diff' better in future.
Thanks for raising this.